src/main/java/nl/tudelft/context/model/graph/GraphNode.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.context.model.graph;
+
+import nl.tudelft.context.controller.GraphController;
+import nl.tudelft.context.controller.MainController;
+import nl.tudelft.context.drawable.DefaultLabel;
+import nl.tudelft.context.drawable.DrawableNode;
+import nl.tudelft.context.drawable.SinglePointLabel;
+
+/**
+ * @author René Vennik <renevennik@gmail.com>
+ * @version 1.0
+ * @since 1-6-2015
+ */
+public class GraphNode extends DefaultNode {
+
+    /**
+     * Counter for new id.
+     */
+    public static int counter = -1;
```

Comment:
*Auto-generated comment*

    Variable 'counter' must be private and have accessor methods.